u8timeseries/timeseries.py

```diff
@@ -458,22 +447,36 @@ def shift(self, n: int) -> 'TimeSeries':
         new_time_index = self._series.index.map(lambda ts: ts + n * self.freq())
         new_series = self._series.copy()
         new_series.index = new_time_index
-        new_conf_lo = None
-        new_conf_hi = None
-        if self._confidence_lo is not None:
-            new_conf_lo = self._confidence_lo.copy()
-            new_conf_lo.index = new_time_index
-        if self._confidence_hi is not None:
-            new_conf_hi = self._confidence_hi.copy()
-            new_conf_hi.index = new_time_index
-        return TimeSeries(new_series, new_conf_lo, new_conf_hi)
+        return TimeSeries(new_series)
+
```

Comment:
How about a `stack` function to stack several (multi or uni- variate) series into one?

Reply:
done in step 2